sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1506,6 +1533,46 @@ def process_bundle(self, inputs, expected_outputs):
     return result, split_results
 
 
+class ParallelBundleManager(BundleManager):
+
+  def __init__(
+      self, controller, get_buffer, get_input_coder_impl, bundle_descriptor,
+      num_workers, progress_frequency=None, skip_registration=False):
```

Comment:
Generally it'd be preferable to pass num_workers as the last argument, by keyword, so the signatures between super and subclasses agree.